test/mission_control/jobs/workers_relation_test.rb

```diff
@@ -0,0 +1,17 @@
+require "test_helper"
+
+class MissionControl::Jobs::WorkersTest < ActiveSupport::TestCase
+  setup do
+    queue_adapter = ActiveJob::QueueAdapters::SolidQueueExt
+    @workers_relation = MissionControl::Jobs::WorkersRelation.new(queue_adapter:)
```

Comment:
```suggestion
    @workers_relation = MissionControl::Jobs::WorkersRelation.new(queue_adapter: queue_adapter)
```
so that it works with Ruby 3.0.